From: Mustafa Çalışkan <musfay@protonmail.com>
Date: Wed, 28 Feb 2021 01:03:23 +0300
Subject: [PATCH] change test dirs

---
 configuration/configuration.go | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/configuration/configuration.go b/configuration/configuration.go
index 01f424c5..d8d4f2bb 100644
--- a/configuration/configuration.go
+++ b/configuration/configuration.go
@@ -157,8 +157,7 @@ func IsBundledInDesktopIDE(settings *viper.Viper) bool {
 	// To determine if the CLI is bundled with an IDE, We check an arbitrary
 	// number of folders that are part of the IDE install tree
 	tests := []string{
-		"tools-builder",
-		"examples/01.Basics/Blink",
+		"hardware", // for using platform without IDE
 	}
 
 	for _, test := range tests {
-- 
2.30.1
